index.html

```diff
@@ -130,31 +130,32 @@ <h4>Island Getaway</h4>
         <button type="button" class="btn btn-info">Get More Info</button>
       </div>
     </div>
+    <!-- JavaScript Component: Carousel -->
     <div class="row">
-      <div class="col-sm-12 mt-3">
-          <div id="carouselExampleSlidesOnly" class="carousel slide" data-ride="carousel">
-              <div class="carousel-inner">
-                <div class="carousel-item active">
-                  <img class="d-block w-100" src="../img/slider-1.jpg" alt="First slide">
-                </div>
-                <div class="carousel-item">
-                  <img class="d-block w-100" src="../img/slider-2.jpg" alt="Second slide">
-                </div>
-                <div class="carousel-item">
-                  <img class="d-block w-100" src="../img/slider-3.jpg" alt="Third slide">
-                </div>
-              </div>
+      <div class="col-sm-12 mt-4">
+        <div id="carouselExampleSlidesOnly" class="carousel slide" data-ride="carousel">
+          <div class="carousel-inner">
+            <div class="carousel-item active">
+              <img class="d-block w-100" src="../img/slider-1.jpg" alt="First slide">
```

Comment:
Your img paths are incorrect in your carousel. It should just be img/slider-1.jpg etc. Once that is changed, the pictures should show up in the carousel once viewed.

Reply:
Thank you! Was wondering why that was happening. Thanks alot for the input